governance.md

```diff
@@ -55,8 +55,8 @@ itself. Examples:
 * Horizontal: Scalability, Architecture
 * Project: Testing, Release, Docs, PM, Contributor Experience
 
-SIGs must have at least one and ideally two SIG leads at any given
-time. SIG leads are intended to be organizers and facilitators,
+SIGs must have at least one and ideally two SIG chairs at any given
+time. SIG chairs are intended to be organizers and facilitators,
```

Comment:
The only SIG with multiple chairs/leads from the same company is SIG VMWare. The rest, with multiple chairs, are from differing companies (see the [sig list](https://github.com/kubernetes/community/blob/master/sig-list.md)). When it comes to [community membership](https://github.com/kubernetes/community/blob/master/community-membership.md) we have a sponsor requirement of:

> Sponsors must be from multiple member companies to demonstrate integration across community.

For SIG Chairs do we want something similar?

Reply:
@mattfarina I don't think it's a problem unless the *sig* feels it's a problem.

Reply:
I'm going to leave this to another update to keep this PR from sprawling.  I was just fixing s/leads/chairs/ here.  There are concerns here around how we curate voting members for the steering committee that we are actively working through now.  This is part of that hairball.

Reply:
Regarding the vmware SIG, the intent is that we will actively seek a non vmware employee who is willing and able to assist in SIG leadership roles and responsibilities. The goal is to ensure that this SIG is open to all interested parties. The initial leadership composition is a short term artifact of bootstrapping the SIG. Until, and even after diversity is achieved, the SIG intends to actively engage in recruitment to include users, partners and any others interested.

Reply:
I think that having chairs from multiple companies/organizations should be very strongly encouraged, and that single-company groups of chairs should be the exception, and clearly justified when this is required for practical reasons.